src/test/java/org/apache/maven/model/jdom/it/DependencyManagementEtlIT.java

```diff
@@ -37,13 +37,6 @@ public void addJDomDependency() throws IOException {
     assertTransformation();
   }
 
```

Comment:
Why do you remove this test? I would be nice to know if one could add a dependency as first dependency.

Reply:
It's not actually removed. It has been moved to the super test class and is inherited here.